src/components/forms/VsiForm.js

```diff
@@ -181,15 +192,16 @@ class VsiForm extends Component {
           <IcseSelect
             formName="vsi_form"
             name="encryption_key"
+            className={classNameModalCheck}
             labelText="Encryption Key"
             groups={this.props.encryptionKeys}
             value={this.state.encryption_key}
             handleInputChange={this.handleInputChange}
-            invalid={this.props.invalidCallback(this.state)}
```

Comment:
bring this back or change it to something that checks if null or empty string so it's marked as invalid when it is invalid